deploy/helm/overrides.yaml

```diff
@@ -1,4 +1,58 @@
 prometheus:
   prometheusSpec:
     remoteWrite:
-    - url: http://fluentd:9888/prometheus.metrics
+    - url: http://fluentd:9888/prometheus.metrics.state.daemonset
```

Comment:
Are these new url matching to the source endpoints created by Sam's PR?

Reply:
I don't think these urls need to match with the source endpoints created by my PR. Since fluentd accepts all of these metrics from prometheus and more or less treats them all the same (since they are all retagged with `prometheus.datapoint`). Then we separate the metrics again by `job` to send to the source endpoints on Sumo's end.

Reply:
yes. they are transformed to same tag in datapoint filter stage so no impact. spliting them into different urls maybe benefit for the statistics later.